.gitignore

```diff
@@ -106,3 +106,9 @@ CMakeSettings.json
 
 # Python pickle files
 *.pkl
+
+# Large fules from partitioner builds
+*.omcp
```

Comment:
aren't these now hdf5 files? So this shouldn't be needed, right?

Reply:
Removed.